sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -49,7 +49,12 @@ export function createFakeApplication(relations?: {
   application.applicationStatus = ApplicationStatus.Submitted;
   application.relationshipStatus = RelationshipStatus.Single;
   application.currentAssessment = relations?.currentStudentAssessment;
-  application.applicationNumber = faker.random.alphaNumeric(10);
+  // Application numbers are expected to be a string of number
+  // with fixed length of 10 characters.
+  application.applicationNumber = faker.random
```

Comment:
I am still not 100% sure about this assumption but apparently is what we got from @JasonCTang, right?
Either way, the 10 digits number will not start with zero (I assume), so it would be better to change the `padStart` to a `padEnd` IMO.

Reply:
apparently is what we got from @JasonCTang, right? - Yes. 
Sure, I can also set the min as 1000000000 in faker.random.number so that it always has 10 numbers.